src/History/components/MyLetter/MyLetter.style.ts

```diff
@@ -1,12 +1,25 @@
 import styled from '@emotion/styled';
 
-export const MyLetterWrapper = styled.article`
+export const MyLetterWrapper = styled.article<{
+  noteTextColor: string;
+  noteBackground: string;
+}>`
   width: 100%;
   height: 16.3rem;
   padding: 1.3rem 1.2rem;
 
   border-radius: 0.4rem;
-  background-color: ${({ theme }) => theme.colors.sub_purple};
+  ${({ noteBackground }) => {
+    if (noteBackground.substring(0, 1) === '#') {
+      return `background-color: ${noteBackground}`;
+    } else {
+      return `background: url(${noteBackground})`;
+    }
+  }};
```

Comment:
P5: 요것도 이렇게 수정해도 될 것 같습니다! 
저는 이게 더 가독성이 좋다고 생각해서 아래처럼 구현하는 편인데, 개인 취향의 영역이니 참고만 해주셔도 됩니당 ~
```suggestion
  ${({ noteBackground }) => {
    noteBackground.substring(0, 1) === '#') ? 
    css`background-color: ${noteBackground}` : css`background: url(${noteBackground})`
  }};
```

Reply:
내부에서 삼항연산자 사용이 안 되는 줄 알았는데..... 확인 곰마워요!!!

Reply:
P1: 삼항 연산자로 바꾸면 에러 난다고 했었는데 요렇게 구현하니까 에러 안나네요~
코드 구조에 이슈가 있었나봐요 아래 코드 한 번 복붙해보세용 !!
```suggestion
${({ noteBackground }) =>
    noteBackground.substring(0, 1) === '#'
      ? css`
          background-color: ${noteBackground};
        `
      : css`
          background: url(${noteBackground});
        `};
```